Album/ViewControllers/ProfileControllers/ProfileViewController.m

```diff
@@ -22,71 +22,71 @@ @interface ProfileViewController () <UICollectionViewDataSource, UICollectionVie
 @implementation ProfileViewController
 
 - (void)viewDidLoad {
-    [super viewDidLoad];
-    // Assign collection view delegate and dataSource
-    self.friendsCollectionView.dataSource = self;
-    self.friendsCollectionView.delegate = self;
-    // Load profile image
-    [self fetchProfile];
+	[super viewDidLoad];
+	// Assign collection view delegate and dataSource
+	self.friendsCollectionView.dataSource = self;
+	self.friendsCollectionView.delegate = self;
+	// Load profile image
+	[self fetchProfile];
 }
 - (IBAction)activitiesButton:(id)sender {
-    [self performSegueWithIdentifier:@"activitiesSegue" sender:nil];
+	[self performSegueWithIdentifier:@"activitiesSegue" sender:nil];
 }
 - (IBAction)addFriendButton:(id)sender {
-    [self performSegueWithIdentifier:@"addFriendSegue" sender:nil];
+	[self performSegueWithIdentifier:@"addFriendSegue" sender:nil];
 }
 - (IBAction)settingsButton:(id)sender {
-    [self performSegueWithIdentifier:@"settingsSegue" sender:nil];
+	[self performSegueWithIdentifier:@"settingsSegue" sender:nil];
 }
 
-- (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section{
-    return 0;
+- (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section {
+	return 0;
 }
 
-- (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollectionViewLayout *)collectionViewLayout sizeForItemAtIndexPath:(NSIndexPath *)indexPath{
-    return CGSizeMake(3, 3);
+- (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollectionViewLayout *)collectionViewLayout sizeForItemAtIndexPath:(NSIndexPath *)indexPath {
+	return CGSizeMake(3, 3);
```

Comment:
3 by 3 pixels?

Reply:
This is a temporary place holder. Will change later when finish implementing profile views.